packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -340,6 +352,44 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
             tooltip: 'Collapse All',
             priority: 1,
         });
+
+        // Register 'more actions' toolbar item (consists of multiple commands for the workspace root).
+        // Commands are available when there is a single-root workspace currently opened.
+        const navigatorRegisterItem = (item: Mutable<TabBarToolbarItem>) => {
+            const commandId = item.command;
+            const id = 'navigator.tabbar.toolbar.' + commandId;
+            const command = this.commandRegistry.getCommand(commandId);
+            this.commandRegistry.registerCommand({ id, iconClass: command && command.iconClass }, {
+                execute: (w, ...args) => w instanceof FileNavigatorWidget
+                    && this.commandRegistry.executeCommand(commandId, ...args),
+                isEnabled: (w, ...args) => w instanceof FileNavigatorWidget
+                    && !!this.workspaceService.workspace && !this.workspaceService.isMultiRootWorkspaceOpened
+                    && this.commandRegistry.isEnabled(commandId, ...args),
+                isVisible: (w, ...args) => w instanceof FileNavigatorWidget
```

Comment:
`isToggled` missing?

Reply:
Thanks, I'll include `isToggled` as well for the toolbar commands.